Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -991,33 +583,35 @@ def _exec_editor_multitest(self, request: _pytest.fixtures.FixtureRequest,
             if temp_batched_file:
                 os.unlink(temp_batched_file.name)
         return results
-    
+
     def _run_single_test(self, request: _pytest.fixtures.FixtureRequest,
                          workspace: ly_test_tools._internal.managers.workspace.AbstractWorkspaceManager,
                          editor: ly_test_tools.launchers.platforms.base.Launcher,
-                         editor_test_data: TestData, test_spec: EditorSingleTest) -> None:
+                         collected_test_data: EditorTestSuite.TestData,
+                         test_spec: EditorSingleTest) -> None:
         """
         Runs a single test (one editor, one test) with the given specs
         :request: The Pytest Request
         :workspace: The LyTestTools Workspace object
         :editor: The LyTestTools Editor object
-        :editor_test_data: The TestData from calling editor_test_data()
+        :collected_test_data: The TestData from calling collected_test_data()
         :test_spec: The test class that should be a subclass of EditorSingleTest
         :return: None
         """
-        self._setup_editor_test(editor, workspace, editor_test_data)
+        self._setup_test_run(workspace, collected_test_data)
+        editor.configure_settings()
         extra_cmdline_args = []
         if hasattr(test_spec, "extra_cmdline_args"):
             extra_cmdline_args = test_spec.extra_cmdline_args
 
-        result = self._exec_editor_test(request, workspace, editor, 1, "editor_test.log", test_spec, extra_cmdline_args)
+        result = self._exec_editor_test(request, workspace, editor, 1, LOG_NAME, test_spec, extra_cmdline_args)
```

Comment:
this looks good, but remember to use the argument inside the function and not the constant